packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -215,16 +215,36 @@
       </div>
       <div class="row">
         <div class="field-wrapper">
-          <a class="btn social-btn" :href="uris.github">
+          <input
+            type="button"
+            id="continue-github"
+            value="CONTINUE WITH GITHUB"
+            class="btn social-btn"
+            @click="
+              socialUrl = uris.github;
+              isShowTermsConditionsDialog = true;
+            "
```

Comment:
Can we move the modal opening and closing code into methods?

Reply:
if you meant to create open/close methods for `DialogWindow` component then we probably need to create that component programmatically by using `new` for example and then it will be possible to call open/close methods from outside or I didn't get you? @francoislehoux-okta

Reply:
^This

Reply:
done